doc/nrf/shortcuts.txt

```diff
@@ -86,3 +86,5 @@
 .. |vsc_extension_instructions| replace:: The `nRF Connect for Visual Studio Code`_ extension is a complete IDE for developing applications for nRF91, nRF53 and nRF52 Series devices. This includes an interface to the compiler and linker, an RTOS-aware debugger, a seamless interface to the nRF Connect SDK, and a serial terminal.
 
 .. |VSC| replace:: nRF Connect for Visual Studio Code
+
+.. |thingy53_sample_note| replace:: If you build this application for Thingy:53, it enables additional features proper of Thingy:53-compatible application. See :ref:`thingy53_app_guide` for details.
```

Comment:
@mia-ko 

> proper of -> proper to?
> As in appropriate to/belonging to.

Maybe it might be better just to cut the end off?

```suggestion
.. |thingy53_sample_note| replace:: If you build this application for Thingy:53, it enables additional features. See :ref:`thingy53_app_guide` for details.
```

Reply:
applied

Reply:
> @mia-ko
> 
> > proper of -> proper to?
> > As in appropriate to/belonging to.
> 
> Maybe it might be better just to cut the end off?

Agree.